scripts/install.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/sh
+#
+# This script installs the latest release of Tenzir.
+#
+
+# Treat unset variables as an error when substituting.
+set -u
+
+#
+# Helper utilities
+#
+
+# Only use colors when we have a TTY.
+if [ -t 1 ]
+then
+  escape() { printf "\033[%sm" "$1"; }
+else
+  escape() { :; }
+fi
+make_bold() { escape "1;$1"; }
+blue="$(make_bold 34)"
+bold="$(make_bold 39)"
+normal="$(escape 0)"
+
+# Prints an "action", i.e., something the script is doing.
+# Use this to transparently inform the user about the key
+# steps this script takes.
+action() {
+  printf "${blue}==>${bold} %s${normal}\n" "$@"
+}
+
+#
+# Execution
+#
+
+if [ -z "${BASH_VERSION:-}" ]
+then
+  echo "Bash is required to interpret this script."
+  exit 1
+fi
+
+# Print welcome banner.
+echo "${bold}${blue}"
+echo "                _____ _____ _   _ ________ ____  "
+echo "               |_   _| ____| \ | |__  /_ _|  _ \ "
+echo "                 | | |  _| |  \| | / / | || |_) |"
+echo "                 | | | |___| |\  |/ /_ | ||  _ < "
+echo "                 |_| |_____|_| \_/____|___|_| \\_\\"
+echo "${normal}"
+echo "                            INSTALLER"
+echo
+
+# Detect OS.
+action "Identifying platform"
+PLATFORM=
+OS=$(uname -s)
+if [ "${OS}" = "Linux" ]
+then
+    if [ -f "/etc/debian_version" ]
+    then
+        PLATFORM=Debian
+    elif [ -f "/etc/NIXOS" ]
+    then
+        PLATFORM=NixOS
+    else
+        PLATFORM=Linux
+    fi
+elif [ "${OS}" = "Darwin" ]
+then
+  PLATFORM=macOS
+fi
+echo "Found ${PLATFORM}"
+
+# Select appropriate package.
+action "Identifying package"
+PACKAGE=
+if [ "${PLATFORM}" = "Debian" ]
+then
+  PACKAGE="tenzir-linux-static.deb"
+elif [ "${PLATFORM}" = "Linux" ]
+then
+  PACKAGE="tenzir-linux-static.tar.gz"
+elif [ "${PLATFORM}" = "NixOS" ]
+then
+  echo "Try Tenzir with our ${bold}flake.nix${normal}:"
+  echo
+  echo "    ${bold}nix run github:tenzir/tenzir/stable${normal}"
```

Comment:
The https://github.com/tenzir/tenzir/tree/stable branch is 500+ commits behind, what is our plan with that @tobim ?

Also, is our cachix working? I tried 

```nix run github:tenzir/tenzir/stable```

but it is building stuff from source. (I have vast cachix enbled).

Reply:
It sounds like we want a story for this, to be addressed post alpha testing. As long as the instructions are sound, we can focus on the implementation of this separately.

Reply:
Cache not working needs a story, yes. But the correct release binary should be part of alpha.

Reply:
I would assume that the cache is broken because of the repository-renaming.

We usually update the `stable` branch when we cut a release. So at least for the Alpha, we would need a different mechanism here.

Reply:
> Also, is our cachix working? I tried
>
> nix run github:tenzir/tenzir/stable
>
> but it is building stuff from source. (I have vast cachix enbled).

We can't use cachix for closed source code, since we are still on the free version. (poke @mavam)

Reply:
I'm resolving this thread, as the Cachix discussion takes place elsewhere.